examples/iscsi-dd.c

```diff
@@ -32,22 +32,23 @@ const char *initiator = "iqn.2010-11.ronnie:iscsi-inq";
 uint32_t max_in_flight = 50;
 uint32_t blocks_per_io = 200;
 
+struct src_dst {
```

Comment:
How about renaming 'struct src_dst' into 'struct iscsi_endpoint'? Otherwise this patch looks good to me.

Reply:
That's a much nicer name, thanks.